python/ray/train/base_trainer.py

```diff
@@ -278,7 +365,7 @@ def setup(self) -> None:
         """
         pass
 
-    def preprocess_datasets(self) -> None:
+    def preprocess_datasets(self, should_fit_preprocessor: bool = True) -> None:
```

Comment:
Can we shorten this name? I think this is expressive enough
```suggestion
    def preprocess_datasets(self, fit_preprocessor: bool = True) -> None:
```